x-pack/plugins/enterprise_search/public/applications/enterprise_search_content/components/search_index/pipelines/ml_inference/model_select_option.tsx

```diff
@@ -49,7 +55,9 @@ export const MlModelSelectOption: React.FC<MlModelSelectOptionProps> = ({ model
             <EuiFlexGroup gutterSize="xs">
               <EuiFlexItem>
                 <span>
-                  <EuiBadge color="hollow">{type}</EuiBadge>
+                  <EuiBadge color={isTextExpansionModel(model) ? 'success' : 'hollow'}>
```

Comment:
This was in the AC for another issue so I handled this in #153901

Reply:
I introduced a component for it since we need to do this here and in the `inference_pipeline_card` [see this commit](https://github.com/elastic/kibana/pull/153901/commits/158b466b5fbc330d675c70eba3a87c8095826102)

I do like that your added the `TEXT_EXPANSION_FRIENDLY_TYPE` constant vs how I just hard-coded it in the component.